frontend/lib/requests/highlights.ts

```
@@ -78,3 +83,50 @@ export async function getHighlightsByDroplet(
+    `${STRAPI_API_URL}/api/highlights?filters[lesson][id][$eq]=${lessonId}&filters[authorized_user][id][$eq]=${authorizedUser.id}`,
+    {
+      headers: {
+        "Content-Type": "application/json",
+        Authorization: `Bearer ${STRAPI_ACCESS_TOKEN}`,
+      },
+    },
+  );
+  return response.json();
+}
+
+export async function createHighlight(highlightData: any) {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)